app/views/templates/main_template.scala.html

```diff
@@ -40,7 +40,7 @@
                 timeout: 900,
                 countdown: 120,
                 keep_alive_url: "@controllers.timeout.routes.SessionTimeoutController.keepAlive().url",
-                title: "@messages("timeoutDialog.title")",
+                title: "@messages("timeoutDialog.title") + " " + @messages("titlePattern.serviceName.govUk")",
```

Comment:
I don't think this is necessary, I think the "title" for the timeout is actually the heading of the popup that appears to users, in which case, we likely do not want the service name and gov.uk appended

Reply:
done